_data/documentation.yaml

```
@@ -0,0 +1,78 @@
+- header: Overview
+  pages:
+  - title: Language refrence (TSPL)
+    url: /documentation/tspl/
+    description: |
+      The Swift Programming Language book is the authoritative reference for Swift,
```

```suggestion
      The Swift Programming Language (TSPL) book is the authoritative reference for Swift,
```
Usually an abbreviation is defined immediately following the first use of the full term. Moving the abbreviation here instead of in the title.